metadata-ingestion/src/datahub/ingestion/source/looker/lookml_source.py

```diff
@@ -2163,36 +960,47 @@ def get_internal_workunits(self) -> Iterable[MetadataWorkUnit]:  # noqa: C901
                                 else project_name
                             )
 
-                            # if project is base project then it is available as self.base_projects_folder[_BASE_PROJECT_NAME]
+                            # if project is base project then it is available as self.base_projects_folder[
+                            # _BASE_PROJECT_NAME]
                             base_folder_path: str = str(
                                 self.base_projects_folder.get(
                                     current_project_name,
                                     self.base_projects_folder[_BASE_PROJECT_NAME],
                                 )
                             )
 
+                            view_context: LookerViewContext = LookerViewContext(
+                                raw_view=raw_view,
+                                view_file=looker_viewfile,
+                                view_connection=connectionDefinition,
+                                view_file_loader=viewfile_loader,
+                                looker_refinement_resolver=looker_refinement_resolver,
+                                base_folder_path=base_folder_path,
+                                reporter=self.reporter,
+                            )
+
                             maybe_looker_view = LookerView.from_looker_dict(
                                 project_name=current_project_name,
-                                base_folder_path=base_folder_path,
                                 model_name=model_name,
-                                looker_view=raw_view,
+                                view_context=view_context,
                                 connection=connectionDefinition,
                                 looker_viewfile=looker_viewfile,
```

Comment:
`connection` and `looker_view_file`are already available via view_context. Need not pass again.